component-library/src/app/pages/input-documentation/input-documentation.component.scss

```diff
@@ -0,0 +1,182 @@
+@use "../../../../../core-library/util/device" as device;
+
+.content-doc-wrapper {
+  box-sizing: border-box;
+  max-width: 900px;
+
+  img {
+    width: 100%;
+  }
+}
+
+h3 {
+  display: inline-block;
+  font-weight: 700;
+
+  &.primary {
+    font-size: 36px;
+  }
+
+  &.secondary {
+    font-size: 28px;
+    margin-top: 60px;
+  }
+}
+
+.body1.intro {
+  margin-top: 0px;
+  margin-bottom: 0px;
+}
+
+.body3.demo {
+  margin-top: 0px;
+  margin-bottom: 24px;
+}
+
+.body3.type-body {
+  margin-top: 12px;
+  margin-bottom: 12px;
+}
+
+.subtitles {
+  margin-bottom: 12px;
+}
+
+.body3.subtext {
+  margin-top: 12px;
+  margin-bottom: 32px;
+}
+
+.body3.last {
+  margin-top: 12px;
+  margin-bottom: 0px;
+}
+
+.body3.anatomy {
+  margin-bottom: 0px;
+}
+
+
+ul {
+  margin-top: 0px;
+  margin-bottom: 12px;
+
+}
+
+.end-list {
+  margin-bottom: 0px;
+}
+
+>a {
+  margin-left: 16px;
+  position: relative;
+  bottom: 5px;
+  text-decoration: none !important;
```

Comment:
could we scrap the important tag here since this is a new file - ideally we want 0 important tags in project. The other ones should be handled as part of future cleanup so no need to touch those.